src/modules/confirmation/ModuleConfirmation.vue

```diff
@@ -291,6 +313,7 @@ export default {
   mixins: [handlerAnalytics],
   data() {
     return {
+      isLedger: false,
```

Comment:
Remove this and it's instances. You're already checking for if the wallet is ledger via identifier